assets/sass/common/admin/icons/_variables-theme.scss

```diff
@@ -12,6 +12,7 @@ $icon-color-light-green: #b0d244;
 $icon-color-danger: $error-color;
 $icon-color-warning: $warning-color;
 $icon-color-info: $primary-color;
+$icon-color-super-light: #e0e0e0;
```

Comment:
I took the color from the mock in the jira issue.

If I use `#a7a7a7` it looks like:
<img width="404" alt="screen shot 2017-01-11 at 1 07 29 pm" src="https://cloud.githubusercontent.com/assets/17706432/21846248/f57d600e-d7fe-11e6-84cd-24146b316c19.png">
@vincentvoong-okta Is it OK?